sdk/iot/Azure.Iot.Hub.Service/samples/IotHubClientSamples/MehodInvocationSamples.cs

```diff
@@ -0,0 +1,236 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Iot.Hub.Service.Models;
+
+namespace Azure.Iot.Hub.Service.Samples
+{
+    /// <summary>
+    /// This sample shows how to invoke a method on a device and module.
+    /// This sample requires the device sample to be running so that it can connect to the device.
```

Comment:
Note that the device client sample must also have subscribed to device methods. Simply being online isn't enough

Reply:
I agree. We should write that in the device sample when we add them.